client/my-sites/marketing/connections/service-examples.jsx

```diff
@@ -286,12 +286,35 @@ class SharingServiceExamples extends Component {
 		return [
 			{
 				image: {
-					src: '/calypso/images/sharing/slack-screenshot.png',
-					alt: this.props.translate( 'Get slack notifications on new P2 posts.', {
+					src: '/calypso/images/sharing/slack-screenshot-1.png',
+					alt: this.props.translate( 'Get Slack notifications with every new P2 post.', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.